README.md

```diff
@@ -6,51 +6,43 @@ We run an [internal server](./internal-server.md) so private datasets can be kep
 
 We have have general experience [tracking neuroimaging data with git-annex](./git-annex.md).
 
-The data curation scripts should be placed inside the BIDS datasets, under the `code/` folder. See [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code). For more convenience, you can create a PR with a curation script that you are working on, so that others can give feedback; once the script is validated, you can simply close the PR and delete the branch without merging.
+# ⚠️ Every dataset should have the following files 
 
-Every dataset should have a README inside it. Example:
-~~~
-# <NAME OF DATASET>
-
-<PLEASE MODIFY>This is an MRI dataset acquired in the context of the XX project. It also contains manual segmentation of MS lesions from expert raters.
-
-Dataset shared by: <NAME AND EMAIL>
-<IF THERE WAS EMAIL COMM>Email communication: <DATE OF EMAIL AND SUBJECT>
-<REPOSITORY OF PROJECT/MODEL, etc>Repository: https://github.com/ivadomed/model_seg_ms_mp2rage
+```
+├── README.md
+├── participants.json
+├── participants.tsv
+├── dataset_description.json
+└── code
+    └── <script_you_used_for_dataset_curation>.py
+```
 
-<IF THERE ARE UPDATES IN THE DATASET:>2022-10-30: Added new data from INsIDER_SCT_Segmentations_COR
+For details, see [BIDS specification](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#code).
 
-## dataset structure
+## `README.md`
 
-participants.tsv: Include the following columns:
-- 'pathology': can take values listed in https://docs.google.com/spreadsheets/d/1yjcA8Z0COn4OZxusIDHjStH2DpeXvscsj-aWE2X-_sg/edit#gid=0
-- 'data_id' is the name center_study_subject used to locate the unprocessed data as under duke/mri/.
-- 'institution_id' is used to identify CenterStudy.
-- others? XXX (Jan pls continue)
+The `README.md` is a [markdown](https://markdown-guide.readthedocs.io/en/latest/index.html) file describing the dataset in more detail.
 
+<details><summary>README.md template:</summary>
 
-XXX Jan contnue -- asju julien to check afterward
+```
+# <NAME OF DATASET>
 
+This is an MRI dataset acquired in the context of the <XYZ> project. 
+It also contains manual segmentation of <MS lesions/tumors/etc> from <one/two/or more> expert raters. 
+Segmentation are located under the derivatives folder.
 
+## contact person
 
-CONVENTION FOR JSON METADATA
-----------------------------
+Dataset shared by: <NAME AND EMAIL>
+<IF THERE WAS EMAIL COMM>Email communication: <DATE OF EMAIL AND SUBJECT>
+<REPOSITORY OF PROJECT/MODEL, etc>Repository: https://github.com/<organization>/<repository_name>
```

Comment:
I am wary of building in the assumption that datasets and analyses are 1:1. From that diagram I keep referencing on https://handbook.datalad.org/en/latest/basics/101-127-yoda.html, a key feature of reproducible science is that datasets can be reanalysed by other parties or reinterpreted, so that there is a tree of derived works.

If we write this line in, it strongly suggests there can only be a linear descent from each dataset. The clever lab member will see they can just delete this line from the README if they are intending to have their work reused widely, and the clever downstream user will see they can just ignore this line, but the confused and overworked will see this and assume each dataset can only be analysed once.